Index: LVM2.2.02.95/doc/example.conf.in
===================================================================
--- LVM2.2.02.95.orig/doc/example.conf.in	2013-03-14 10:13:30.000000000 +0100
+++ LVM2.2.02.95/doc/example.conf.in	2013-03-14 10:13:39.000000000 +0100
@@ -185,7 +185,7 @@
     # to 1, discards will only be issued if both the storage and kernel provide
     # support.
     # 1 enables; 0 disables.
-    issue_discards = 0
+    issue_discards = 1
 }
 
 # This section allows you to configure the way in which LVM selects
