Index: lvm2-2.02.88/doc/example.conf.in
===================================================================
--- lvm2-2.02.88.orig/doc/example.conf.in	2012-03-09 09:53:24.000000000 +0100
+++ lvm2-2.02.88/doc/example.conf.in	2012-03-09 09:53:30.000000000 +0100
@@ -178,7 +178,7 @@
     # to 1, discards will only be issued if both the storage and kernel provide
     # support.
     # 1 enables; 0 disables.
-    issue_discards = 0
+    issue_discards = 1
 }
 
 # This section allows you to configure the way in which LVM selects
