Index: new/debian/control
===================================================================
--- new.orig/debian/control	2011-09-22 10:21:22.000000000 +0200
+++ new/debian/control	2011-09-22 10:21:53.000000000 +0200
@@ -47,7 +47,7 @@
 
 Package: libdevmapper1.02.1
 Section: libs
-Priority: required
+Priority: optional
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, dmsetup (>= ${binary:Version})
 Breaks: lvm2 (<< 2.02.66)
