Index: LVM2.2.02.95/debian/control
===================================================================
--- LVM2.2.02.95.orig/debian/control	2013-03-14 09:22:00.000000000 +0100
+++ LVM2.2.02.95/debian/control	2013-03-14 09:25:34.000000000 +0100
@@ -1,9 +1,8 @@
 Source: lvm2
 Section: admin
 Priority: optional
-Maintainer: Debian LVM Team <pkg-lvm-maintainers@lists.alioth.debian.org>
-Uploaders: Bastian Blank <waldi@debian.org>
-Build-Depends: dpkg-dev (>= 1.16.1~), debhelper (>= 8.1.3~), automake, libreadline-gplv2-dev, libselinux1-dev, libudev-dev, pkg-config
+Maintainer: Proxmox Support Team <support@proxmox.com>
+Build-Depends: dpkg-dev (>= 1.16.1~), debhelper (>= 8.1.3~), automake, libreadline-gplv2-dev, libselinux1-dev, libudev-dev, pkg-config, libcorosync-pve-dev, redhat-cluster-pve-dev, libopenais-pve-dev, quilt
 Standards-Version: 3.9.1
 Homepage: http://sources.redhat.com/lvm2/
 Vcs-Svn: svn://svn.debian.org/pkg-lvm/lvm2/trunk/
@@ -38,7 +37,7 @@
 Package: clvm
 Priority: extra
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, lvm2 (= ${binary:Version}), lsb-base, corosync | cman
+Depends: ${shlibs:Depends}, ${misc:Depends}, lvm2 (= ${binary:Version}), lsb-base, redhat-cluster-pve
 Description: Cluster LVM Daemon for lvm2
  This package provides the clustering interface for lvm2, when used with
  Red Hat's "cman" or corosync based (eg Pacemaker) cluster infrastructure.
