Index: lvm2-2.02.88/debian/control
===================================================================
--- lvm2-2.02.88.orig/debian/control	2011-12-09 06:23:56.000000000 +0100
+++ lvm2-2.02.88/debian/control	2011-12-09 06:29:54.000000000 +0100
@@ -46,7 +46,7 @@
 
 Package: libdevmapper1.02.1
 Section: libs
-Priority: required
+Priority: optional
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, dmsetup (>= ${binary:Version})
 Breaks: lvm2 (<< 2.02.66)
