Index: new/scripts/lvm2_cluster_activation_systemd_red_hat.service.in
===================================================================
--- new.orig/scripts/lvm2_cluster_activation_systemd_red_hat.service.in
+++ new/scripts/lvm2_cluster_activation_systemd_red_hat.service.in
@@ -9,7 +9,7 @@ Conflicts=shutdown.target
 [Service]
 Type=simple
 RemainAfterExit=yes
-EnvironmentFile=-@sysconfdir@/sysconfig/clvmd
+EnvironmentFile=-/etc/default/clvmd
 ExecStart=@systemdutildir@/lvm2-cluster-activation activate
 ExecStop=@systemdutildir@/lvm2-cluster-activation deactivate
 
Index: new/scripts/lvm2_clvmd_systemd_red_hat.service.in
===================================================================
--- new.orig/scripts/lvm2_clvmd_systemd_red_hat.service.in
+++ new/scripts/lvm2_clvmd_systemd_red_hat.service.in
@@ -13,8 +13,8 @@ Conflicts=shutdown.target
 [Service]
 Type=forking
 Environment=CLVMD_OPTS=-T30
-EnvironmentFile=-@sysconfdir@/sysconfig/clvmd
-ExecStart=@sbindir@/clvmd $CLVMD_OPTS
+EnvironmentFile=-/etc/default/clvmd
+ExecStart=/usr/sbin/clvmd $CLVMD_OPTS
 SuccessExitStatus=5
 TimeoutStartSec=30
 TimeoutStopSec=10
