Index: LVM2.2.02.95/debian/changelog
===================================================================
--- LVM2.2.02.95.orig/debian/changelog	2011-11-26 18:51:44.000000000 +0100
+++ LVM2.2.02.95/debian/changelog	2012-04-27 07:42:36.000000000 +0200
@@ -1,3 +1,41 @@
+lvm2 (2.02.95-1pve2) unstable; urgency=low
+
+  * fix udev rule priorities
+
+ -- Proxmox Support Team <support@proxmox.com>  Fri, 27 Apr 2012 07:42:26 +0200
+
+lvm2 (2.02.95-1pve1) unstable; urgency=low
+
+  * update to latest available lvm 2.0.95
+
+ -- Proxmox Support Team <support@proxmox.com>  Mon, 23 Apr 2012 07:54:36 +0200
+
+lvm2 (2.02.88-2pve2) unstable; urgency=low
+
+  * enable issue_discard in default lvm.conf
+
+ -- Proxmox Support Team <support@proxmox.com>  Fri, 09 Mar 2012 09:56:19 +0100
+
+lvm2 (2.02.88-2pve1) unstable; urgency=low
+
+  * update to 2.02.88-2
+
+ -- Proxmox Support Team <support@proxmox.com>  Fri, 09 Dec 2011 06:50:49 +0100
+
+lvm2 (2.02.86-1pve2) unstable; urgency=low
+
+  * disable clvmd by default
+
+  * do not set locking type automatically
+
+ -- Proxmox Support Team <support@proxmox.com>  Fri, 25 Nov 2011 08:45:28 +0100
+
+lvm2 (2.02.86-1pve1) unstable; urgency=low
+
+  * backport to squeeze for use with Proxmox VE
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 20 Sep 2011 12:53:25 +0200
+
 lvm2 (2.02.88-2) unstable; urgency=low
 
   * Enable corosync and openais support. (closes: #563320)
@@ -76,7 +114,7 @@
 
   * Import upstream version 2.02.72:
     - CVE-2010-2526: Fix insecure communication between lvm2 and clvmd.
-     (Closes: #591204)          
+     (Closes: #591204)
     - Only use single node clvm if explicitly requested.
 
  -- Bastian Blank <waldi@debian.org>  Thu, 19 Aug 2010 14:44:02 +0200
@@ -543,7 +581,7 @@
   * Don't make /lib/lvm2 the default library directory as it causes initrd
     troubles.
     Closes: #287042
-  
+
  -- Patrick Caulfield <patrick@debian.org>  Fri, 24 Dec 2004 12:27:05 +0000
 
 lvm2 (2.00.31-1) unstable; urgency=low
@@ -571,7 +609,7 @@
   * Remove "is not recommended for production use" from package description.
     Closes: #278919
   * Add lvm2create_initrd to documentation directory.
-  
+
  -- Patrick Caulfield <patrick@debian.org>  Wed,  3 Nov 2004 09:37:56 +0000
 
 lvm2 (2.00.24-2) unstable; urgency=low
@@ -616,7 +654,7 @@
     Closes: #260700
   * Add Czech debconf translation
     Closes: #265545
-  
+
  -- Patrick Caulfield <patrick@debian.org>  Fri, 20 Aug 2004 10:44:11 +0100
 
 lvm2 (2.00.19-3) unstable; urgency=low
@@ -756,7 +794,7 @@
  -- Andres Salomon <dilinger@voxel.net>  Thu, 07 Aug 2003 01:04:59 -0400
 
 lvm2 (1.95.15-2) unstable; urgency=low
-  
+
   * Calm-before-the-storm release; the next one will either be 2.00 (once
     the kill-yer-VGs bug is fixed) or 1.95 with lvm10 compatibility (once
     patrick returns from vacation).
@@ -832,7 +870,7 @@
   * Add dmsetup as a suggestion.
   * Add /etc/lvm/lvm.conf conffile.
   * Add undocumented(7) for the commands missing manpages.
-  
+
  -- Andres Salomon <dilinger@mp3revolution.net>  Mon,  4 Mar 2002 04:51:26 -0500
 
 lvm2 (0.95.02cvs20020220) unstable; urgency=low
