Index: new/debian/control
===================================================================
--- new.orig/debian/control	2013-03-29 07:17:35.000000000 +0100
+++ new/debian/control	2013-03-29 07:17:50.000000000 +0100
@@ -1,9 +1,9 @@
 Source: lvm2
 Section: admin
 Priority: optional
-Maintainer: Debian LVM Team <pkg-lvm-maintainers@lists.alioth.debian.org>
+Maintainer: Proxmox Support Team <support@proxmox.com>
 Uploaders: Bastian Blank <waldi@debian.org>
-Build-Depends: dpkg-dev (>= 1.16.1~), debhelper (>= 8.1.3~), automake, libcman-dev (>> 2), libcorosync-dev, libdlm-dev (>> 2), libreadline-gplv2-dev, libselinux1-dev, libudev-dev, openais-dev, pkg-config
+Build-Depends: dpkg-dev (>= 1.16.1~), debhelper (>= 8.1.3~), automake, libreadline-gplv2-dev, libselinux1-dev, libudev-dev, pkg-config, libcorosync-pve-dev, redhat-cluster-pve-dev, libopenais-pve-dev, quilt
 Standards-Version: 3.9.1
 Homepage: http://sources.redhat.com/lvm2/
 Vcs-Svn: svn://svn.debian.org/pkg-lvm/lvm2/trunk/
@@ -38,8 +38,7 @@
 Package: clvm
 Priority: extra
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, lvm2 (= ${binary:Version}), lsb-base
-Recommends: corosync | openais | cman
+Depends: ${shlibs:Depends}, ${misc:Depends}, lvm2 (= ${binary:Version}), lsb-base, redhat-cluster-pve
 Multi-Arch: foreign
 Description: Cluster LVM Daemon for lvm2
  This package provides the clustering interface for lvm2, when used with
