Index: new/debian/changelog
===================================================================
--- new.orig/debian/changelog
+++ new/debian/changelog
@@ -1,3 +1,9 @@
+lvm2 (2.02.116-pve1) unstable; urgency=medium
+
+  * New upstream version.
+
+ -- Proxmox Support Team <support@proxmox.com>  Fri, 27 Feb 2015 11:27:16 +0100
+
 lvm2 (2.02.111-2) unstable; urgency=medium
 
   * Install additional man-pages. (closes: #587277, #760679)
