Index: lvm2-2.02.88/debian/control
===================================================================
--- lvm2-2.02.88.orig/debian/control	2011-12-09 06:16:23.000000000 +0100
+++ lvm2-2.02.88/debian/control	2011-12-09 06:23:56.000000000 +0100
@@ -19,20 +19,6 @@
  volume groups can be allocated to logical volumes, which are accessed as
  regular block devices.
 
-Package: lvm2-udeb
-XC-Package-Type: udeb
-Section: debian-installer
-Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: Linux Logical Volume Manager
- This is a udeb, or a microdeb, for the debian-installer.
- .
- This is LVM2, the rewrite of The Linux Logical Volume Manager.  LVM
- supports enterprise level volume management of disk and disk subsystems
- by grouping arbitrary disks into volume groups. The total capacity of
- volume groups can be allocated to logical volumes, which are accessed as
- regular block devices.
-
 Package: clvm
 Priority: extra
 Architecture: any
@@ -77,20 +63,6 @@
  device-mapper; it allows usage of the device-mapper through a clean,
  consistent interface (as opposed to through kernel ioctls).
 
-Package: libdevmapper1.02.1-udeb
-XC-Package-Type: udeb
-Section: debian-installer
-Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: Linux Kernel Device Mapper userspace library
- This is a udeb, or a microdeb, for the debian-installer.
- .
- The Linux Kernel Device Mapper is the LVM (Linux Logical Volume Management)
- Team's implementation of a minimalistic kernel-space driver that handles
- volume management, while keeping knowledge of the underlying device layout
- in user-space.  This makes it useful for not only LVM, but EVMS, software
- raid, and other drivers that create "virtual" block devices.
-
 Package: dmsetup
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, util-linux (>> 2.16)
@@ -98,20 +70,6 @@
 Description: Linux Kernel Device Mapper userspace library
  The Linux Kernel Device Mapper is the LVM (Linux Logical Volume Management)
  Team's implementation of a minimalistic kernel-space driver that handles
- volume management, while keeping knowledge of the underlying device layout
- in user-space.  This makes it useful for not only LVM, but EVMS, software
- raid, and other drivers that create "virtual" block devices.
- .
- This package contains a utility for modifying device mappings.
-
-Package: dmsetup-udeb
-XC-Package-Type: udeb
-Section: debian-installer
-Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: Linux Kernel Device Mapper userspace library
- The Linux Kernel Device Mapper is the LVM (Linux Logical Volume Management)
- Team's implementation of a minimalistic kernel-space driver that handles
  volume management, while keeping knowledge of the underlying device layout
  in user-space.  This makes it useful for not only LVM, but EVMS, software
  raid, and other drivers that create "virtual" block devices.
Index: lvm2-2.02.88/debian/rules
===================================================================
--- lvm2-2.02.88.orig/debian/rules	2011-12-09 06:22:58.000000000 +0100
+++ lvm2-2.02.88/debian/rules	2011-12-09 06:25:31.000000000 +0100
@@ -59,7 +59,7 @@
 	cd $(DIR); autoreconf
 	touch $@
 
-setup: $(STAMPS_DIR)/setup_deb $(STAMPS_DIR)/setup_udeb
+setup: $(STAMPS_DIR)/setup_deb
 
 $(STAMPS_DIR)/setup_deb: SOURCE_DIR = $(BUILD_DIR)/source
 $(STAMPS_DIR)/setup_deb: DIR = $(BUILD_DIR)/build_deb
@@ -101,7 +101,7 @@
 		--disable-selinux
 	touch $@
 
-build: $(STAMPS_DIR)/build_deb $(STAMPS_DIR)/build_udeb
+build: $(STAMPS_DIR)/build_deb
 
 build_deb: $(STAMPS_DIR)/build_deb
 build_udeb: $(STAMPS_DIR)/build_udeb
@@ -223,7 +223,7 @@
 	+$(MAKE_SELF) install-base
 
 binary-indep:
-binary-arch: $(addprefix install_,libdevmapper libdevmapper-udeb libdevmapper-dev dmsetup dmsetup-udeb liblvm2 lvm2 lvm2-udeb clvm)
+binary-arch: $(addprefix install_,libdevmapper libdevmapper-dev dmsetup liblvm2 lvm2 clvm)
 
 binary: binary-indep binary-arch
 
