Index: new/debian/rules
===================================================================
--- new.orig/debian/rules
+++ new/debian/rules
@@ -171,7 +171,10 @@ install_clvm: export DH_OPTIONS = -pclvm
 install_clvm: DIR = $(BUILD_DIR)/install_deb
 install_clvm: $(STAMPS_DIR)/install_deb
 	+$(MAKE_SELF) install-base-prep INSTALL_DIR='$(DIR)'
-	dh_installinit --no-start -- start 63 S . start 51 0 6 .
+	# do not install clvm.init (we use lvm2-clvmd.service instead)
+	# do not install clvm.defaults (users should use systemd files to
+	# enable clvmd startup and overwrite settings)
+	dh_systemd_enable --no-enable lvm2-cluster-activation.service
 	+$(MAKE_SELF) install-base
 
 install_dmeventd: export DH_OPTIONS = -pdmeventd
